File: content/snippet/code-signal/arcade/intro/eruption-of-light/elections-winners/elections-winners.js
function electionsWinners(votes, k) {
  let maxVote = Math.max(...votes);
  if (k === 0 && votes.filter((vote) => vote === maxVote).length === 1) {
    return 1;
  }
  return votes.filter((vote) => vote + k > maxVote).length;
}

console.log(electionsWinners([1, 1, 1, 1, 1], 1));

File: content/snippet/code-signal/arcade/intro/eruption-of-light/elections-winners/elections-winners.py
def elections_winners(votes, k):
    max_vote = max(votes)

    if k == 0 and votes.count(max_vote) == 1:
        return 1

    return len([vote for vote in votes if vote + k > max_vote])


print(elections_winners([1, 1, 1, 1, 1], 1))
